packages/@aws-cdk/aws-redshift/lib/private/database-query-provider/table.ts

```diff
@@ -162,3 +181,10 @@ async function updateTable(
 function getSortKeyColumnsString(sortKeyColumns: Column[]) {
   return sortKeyColumns.map(column => column.name).join();
 }
+
+function getEncodingColumnString(column: Column): string {
+  if (column.encoding) {
+    return ` ENCODE ${column.encoding}`;
```

Comment:
minor, but could the space not be included? eg
```suggestion
    return `ENCODE ${column.encoding}`;
```